app/controllers/deep_links_controller.rb

```diff
@@ -9,7 +9,7 @@ def aasa
     supported_apps << "R9SWHSQNV8.to.dev.ios" if SiteConfig.dev_to?
     render json: {
       applinks: {
-        apps: supported_apps,
+        apps: [],
         details: supported_apps.map { |app_id| { appID: app_id, paths: ["/*"] } }
```

Comment:
This change is inspired by research over how other high profile apps have configured their `aasa` (facebook, tiktok, instagram, reddit, etc)

Reply:
For the curious, can you elaborate a bit what this change means for users?

Reply:
I would probably add a comment here explaining why it's empty

Reply:
This is a messy one, I wish I had a better answer but I don't.

In short: Apple's documentation sometimes contradicts itself (or it's just ambiguous at times). For example, the `aasa` JSON structure in [this page](https://developer.apple.com/library/archive/documentation/General/Conceptual/AppSearch/UniversalLinks.html) from **listing 6-1** differs from what's mentioned in [this other page](https://developer.apple.com/documentation/Xcode/supporting-associated-domains).

Universal Links were working for us the way they're currently setup, but this change tries to stick with the structure in the first one of the two options mentioned above. This is the `aasa` structure the high profile apps I originally mentioned rely on. These are the `aasa` files from [fb](https://facebook.com/.well-known/apple-app-site-association), [twitter](https://twitter.com/.well-known/apple-app-site-association), [tiktok](https://tiktok.com/.well-known/apple-app-site-association), [reddit](https://reddit.com/.well-known/apple-app-site-association) & [instagram](https://instagram.com/.well-known/apple-app-site-association).

I added an inline comment with a link to the specific documentation page which should explain why it's empty. Hopefully we don't have to continue to tweak this, but it's possible we'll have to revisit this JSON structure in case we notice an unexpected effect in the behavior (sadly we do need a bit or trial and error to make sure everything works as expected).